internal/security/secretstore/init.go

```diff
@@ -373,12 +373,12 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, _ *sync.WaitGroup, _ s
 		lc.Info("Generating new password for Redis DB")
 		redis5Password, err := cred.GeneratePassword(ctx)
```

Comment:
let's also modify all the `redis5` variables from the source code
thanks

Reply:
`redis5Password` -> `redisPassword`

Reply:
@cloudxxx8 updated!

Reply:
thanks, please also replace the `redis5Pair` in this file with `redisPair`

Reply:
@cloudxxx8 updated! please review again.